client/app/cdn/dedicated/manage/cdn-dedicated-manage.controller.js

```diff
@@ -20,9 +20,9 @@ angular.module("App").controller("CdnManageCtrl", class CdnManageCtrl {
         return this.$q.allSettled([
             this.Cdn.getSelected(this.$stateParams.productId, true),
             this.Cdn.getServiceInfos(this.$stateParams.productId)
-        ]).then((data) => {
-            this.cdn = data[0];
-            this.serviceInfos = data[1];
+        ]).then(([selected, serviceInfos]) => {
```

Comment:
This works ???

Reply:
Destructuring, for sure.

Reply:
In an array ? Not in an object ? I've never seen this before